App.js

```diff
@@ -31,7 +31,7 @@ const App: () => Node = () => {
     async function initializeClient() {
       let ldClient = new LDClient();
       let config = { mobileKey, debugMode: true, evaluationReasons: true };
-      let user = { key: 'test-key', anonymous: true };
+      let user = { kind: 'user', key: 'test-key' };
```

Comment:
The original was anonymous.

Reply:
(There may not have been a good reason, just acknowledging this is different versus just an update.)

Reply:
I don't know if is there any real reason we did this before - I don't see this in our other example apps. (e.g. https://github.com/launchdarkly/hello-js/blob/main/index.html)

And given `anonymous` has an inconsistency between Android and iOS that we are looking to fix, not having people trying it out by default seems to be a better idea.